js/ez-platform-ui-app.js

```diff
@@ -289,6 +290,21 @@
         }
 
         /**
+         * @param {Response} response
+         *
+         * @return {Response}
+         */
+        _updateSfToolbar(response) {
+            var sfToolbar = this.parentNode.querySelector('.sf-toolbar');
```

Comment:
nice :)
but I would not do that here. The fact that you have to use `parentNode` from the app component is a good hint that it's probably not the app responsibility to update the SF toolbar.

IMHO, instead, the code updating the toolbar should be generated in layout.html.twig (only when the toolbar is available in the page if possible, this avoids some checks in JS) and the toolbar should be updated thanks to the `ez:app:updated` event dispatched after each app update. But before being able to do that, we need to enrich the `ez:app:updated` event details to contain the response (at the moment, the `ez:app:updated` has no detail, see [the code dispatching this event](https://github.com/ezsystems/hybrid-platform-ui-core-components/blob/master/js/ez-platform-ui-app.js#L253) vs. [Adding custom data to a CustomEvent](https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Creating_and_triggering_events#Adding_custom_data_%E2%80%93_CustomEvent()))

Then the update code would look like something like:

```js
document.addEventListener('ez:app:updated', function (e) {
    Sfjs.load(
        document.querySelector('.sf-toolbar').id,
        '/_wdt/' + e.details.response.headers.get('x-debug-token')
    );
});
```